model/client4.go

```diff
@@ -3782,10 +3782,17 @@ func (c *Client4) GetChannelsForScheme(schemeId string, page int, perPage int) (
 
 // UploadPlugin takes an io.Reader stream pointing to the contents of a .tar.gz plugin.
 // WARNING: PLUGINS ARE STILL EXPERIMENTAL. THIS FUNCTION IS SUBJECT TO CHANGE.
-func (c *Client4) UploadPlugin(file io.Reader) (*Manifest, *Response) {
+func (c *Client4) UploadPlugin(file io.Reader, force bool) (*Manifest, *Response) {
```

Comment:
I'm not 100% on this, but I think this is a backwards incompatible change at the compilation level -- i.e. plugin authors upgrading will find their scripts broken. 0/5 on if we want to avoid this, but we could expose a `UploadPluginForced` for now and unify on 6.0.

Reply:
Would you find it acceptable if I changed the func signature to `UploadPlugin(file io.Reader, opts ...bool)`, or even `UploadPlugin(file io.Reader, opts ...func ())`. 

Side note: it seems unfortunate that plugins have access to the go Client package, seems like they should be restricted to p.APIs and the http APIs.

Reply:
Never mind above, would be premature. I added a new (wrapper) method `UploadPluginForced`as you suggested.